### PR TITLE
Feat/LIVE-3831 - NFT Gallery - Wallet navigator header animation

### PR DESCRIPTION
**Pull request by nparigi-ledger**
### 📝 Description

NFT Gallery - Wallet navigator header animation

### ❓ Context

- **Impacted projects**: `` live-mobile
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/browse/LIVE-3831

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/89014981/196407256-ce9748e3-88b9-4a95-81d8-1141e1ca4498.mp4

With Reborn (read only) :

https://user-images.githubusercontent.com/89014981/196407244-609400ef-4b53-4674-ac6a-df1502c3910b.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderFlatList.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  const windowHeight = Dimensions.get("window").height;
+  const route = useRoute();
+
+  const onMomentumScrollEnd = useCallback(() => {
+    syncScrollOffset(route.name);
+  }, [route.name, syncScrollOffset]);
+
+  const onScrollEndDrag = useCallback(() => {
+    syncScrollOffset(route.name);
+  }, [route.name, syncScrollOffset]);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderFlatList.tsx:29]: They do the same thing right?

File: apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderScrollView.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  const windowHeight = Dimensions.get("window").height;
+  const route = useRoute();
+
+  const onMomentumScrollEnd = useCallback(() => {
+    syncScrollOffset(route.name);
+  }, [route.name, syncScrollOffset]);
+
+  const onScrollEndDrag = useCallback(() => {
+    syncScrollOffset(route.name);
+  }, [route.name, syncScrollOffset]);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderScrollView.tsx:28]: Same here

File: apps/ledger-live-mobile/src/components/WalletTab/WalletTabHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      style={[
+        {
+          top: 0,
+          height: headerHeight,
+          width: "100%",
+          position: "absolute",
+        },
+        { transform: [{ translateY: y }] },
+      ]}
+    >
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/WalletTab/WalletTabHeader.tsx:25]: can't use `styled` like `Skelleton`?

File: apps/ledger-live-mobile/src/screens/Portfolio/index.tsx [hunk 4]
```diff
@@ -244,39 +245,33 @@ function PortfolioScreen({ navigation }: Props) {
...
+      <BackgroundGradient
+        currentPositionY={currentPositionY}
+        graphCardEndPosition={graphCardEndPosition}
+      />
+      <CollapsibleHeaderFlatList
+        data={data}
+        renderItem={({ item }: { item: React.ReactNode }) => item}
+        keyExtractor={(_: any, index: number) => String(index)}
+        showsVerticalScrollIndicator={false}
+        // onScroll={handleScroll}
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/screens/Portfolio/index.tsx:267]: Remove ?

File: apps/ledger-live-mobile/src/components/WalletTab/WalletTabHeader.tsx (hunk 3 above)
Comment (nparigi-ledger) [reply apps/ledger-live-mobile/src/components/WalletTab/WalletTabHeader.tsx:25]: not sure it is useful, as i will not be used elsewhere, i will still need to inject style inside style prop for the animation, and this Header will never have style props ...

File: apps/ledger-live-mobile/src/screens/Portfolio/index.tsx (hunk 4 above)
Comment (nparigi-ledger) [reply apps/ledger-live-mobile/src/screens/Portfolio/index.tsx:267]: yes !

Comment (nparigi-ledger) [reply apps/ledger-live-mobile/src/screens/Portfolio/index.tsx]: done

File: apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderFlatList.tsx (hunk 1 above)
Comment (nparigi-ledger) [reply apps/ledger-live-mobile/src/components/WalletTab/CollapsibleHeaderFlatList.tsx]: done thanks

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**